ome_zarr/reader.py

```diff
@@ -216,13 +216,22 @@ def __init__(self, node: Node) -> None:
                 except Exception as e:
                     LOGGER.error(f"invalid color - {color}: {e}")
 
+        properties: Dict[int, Dict[str, str]] = {}
+        props_list = image_label.get('properties', [])
+        if props_list:
+            for props in props_list:
+                label_val = props['label-value']
+                del props['label-value']
+                properties[label_val] =  props
```

Comment:
Could you explain what's happening here? We'll need to check whether the underlying dictionary is a deep-copy of the original, or whether it returns a reference which means this may update the properties seen by other code.

Reply:
Yes, if I do `image_label.get('properties', [])` after this `for` loop, I get the properties without the `label-value`s so you probably should create a copy of the `props` before editing.
This seems to work:
```
                properties[label_val] = dict(props)
                del properties[label_val]['label-value']
```